Lib/test/test_pyrepl/test_unix_console.py

```diff
@@ -2,11 +2,18 @@
 import os
 import sys
 import unittest
+import errno
```

Comment:
nit: This import list is alphabetical for the `from` imports, would be nice for these new imports to be added to the right alphabetical places.

Reply:
copy